Mailnag/backends/local.py

```diff
@@ -33,7 +33,7 @@ class MBoxBackend(MailboxBackend):
 	"""Implementation of mbox mail boxes."""
 	
 	def __init__(self, name = '', path=None, **kw):
-		"""Initialize mbox mailbox backens with a name and path."""
+		"""Initialize mbox mailbox backend with a name and path."""
 		self.name = name
```

Comment:
I guess all backends should mark their fields as private with a leading underscore (e.g. self._name)